platform/linux-generic/arch/aarch64/odp_sysinfo_parse.c

```diff
@@ -120,6 +120,10 @@ static void aarch64_part_info(char *str, int maxlen, odp_cpu_arch_arm_t *cpu_isa
 			snprintf(str, maxlen, "Cortex-A77");
 			*cpu_isa = ODP_CPU_ARCH_ARMV8_2;
 			return;
+		case 0xd40:
+			snprintf(str, maxlen, "Neoverse V1");
+			*cpu_isa = ODP_CPU_ARCH_ARMV8_4;
```

Comment:
"Armv8.1-8.4 Extensions without Secure-EL2", right?

Reply:
V1 reference manual lists:
A1.2 Features
Core features:
• Full implementation of the Armv8.4-A A64, A32, and T32 instruction sets excluding the following features:
— Armv8.4-A secure EL2.
— Armv8.4-A small page table support.
— Armv8.4-A Outer Shareable TLB Maintenance Operation (TMO)/TMO-by-range
• Cache clean to Point of Deep Persistence introduced in the Armv8.5‑A extension.
• The random number instructions introduced in the Armv8.5‑A extension.
• The Enhanced Pointer Authentication, excluding the optional FPAC extension introduced in the Armv8.6‑A extension.
• The BFloat16 extension introduced in the Armv8.6‑A extension.
• The Int8 matrix multiply instructions introduced in the Armv8.6‑A extension.
• The Data Gathering Hint instruction introduced in the Armv8.6‑A extension.
...

Maybe v8.4 is close enough? V1 is v8.4 minus couple of features, plus couple of features from v8.5 and v8.6.

Reply:
Yes, in practice this is probably fine, and maybe some other part infos already make a similar approximation. It's just that since the API doesn't explicitly say anything, someone could take the ISA version as a guarantee.